### PR TITLE
feat/account create

### PR DESCRIPTION
**Pull request by domenicsim1**

### CODE REVIEW

File: pkg/cmd/account/azure/create/create.go [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+	opts := &CreateOptions{
+		Ask:     f.Ask,
+		Spinner: f.Spinner(),
+	}
+	descriptionFilePath := ""
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates an azure account",
+		Long:  "Creates an azure account in an instance of Octopus Deploy.",
```
Comment (jbristowe) [inline pkg/cmd/account/azure/create/create.go:56]: `azure` ➡️ `Azure`

File: pkg/cmd/account/azure/create/create.go [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+				Message: "Azure Environment",
+				Options: envMapKeys,
+				Default: "Global Cloud (Default)",
+			}, &opts.AzureEnvironment); err != nil {
+				return err
+			}
+			opts.AzureEnvironment = azureEnvMap[opts.AzureEnvironment]
+			if opts.ADEndpointBaseUrl == "" {
+				if err := opts.Ask(&survey.Input{
+					Message: "AD Endpoint Base Uri",
```
Comment (jbristowe) [inline pkg/cmd/account/azure/create/create.go:280]: `AD Endpoint Base Uri` ➡️ `Active Directory endpoint base URI`

File: pkg/cmd/account/azure/create/create.go [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+					Default: azureADEndpointBaseUri[opts.AzureEnvironment],
+					Help:    "Set this only if you need to override the default Active Directory Endpoint. In most cases you should leave the pre-populated value as is.",
+				}, &opts.ADEndpointBaseUrl); err != nil {
+					return err
+				}
+			}
+			if opts.RMBaseUri == "" {
+				if err := opts.Ask(&survey.Input{
+					Message: "Resource Management Base Uri",
```
Comment (jbristowe) [inline pkg/cmd/account/azure/create/create.go:289]: `Uri` ➡️ `URI`

File: pkg/cmd/account/azure/list/list.go [hunk 4]
```diff
@@ -62,23 +62,24 @@ func listAzureAccounts(client *client.Client, cmd *cobra.Command, s *spinner.Spi
...
 				SubscriptionNumber: acc.SubscriptionID.String(),
-				TenantId:           acc.TenantID.String(),
-				ClientId:           acc.ApplicationID.String(),
+				AccountType:        string(acc.AccountType),
 				AzureEnvironment:   acc.AzureEnvironment,
 			}
 		},
 		Table: output.TableDefinition[accounts.IAccount]{
-			Header: []string{"NAME", "SUBSCRIPTION ID", "TENANT ID", "APPLICATION ID", "AZURE ENV"},
+			Header: []string{"NAME", "SUBSCRIPTION ID", "AZURE ENV"},
```
Comment (jbristowe) [inline pkg/cmd/account/azure/list/list.go:76]: `"AZURE ENV"` ➡️ `"AZURE ENVIRONMENT"` 🤔  Maybe. Not sure about this one.

File: pkg/cmd/account/gcp/create/create.go [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+		Ask:     f.Ask,
+		Spinner: f.Spinner(),
+	}
+	descriptionFilePath := ""
+	keyFilePath := ""
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates an gcp account",
+		Long:  "Creates an Google Cloud Account in an instance of Octopus Deploy.",
```
Comment (jbristowe) [inline pkg/cmd/account/gcp/create/create.go:51]: an ➡️ a

File: pkg/cmd/account/gcp/create/create.go [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+	opts := &CreateOptions{
+		Ask:     f.Ask,
+		Spinner: f.Spinner(),
+	}
+	descriptionFilePath := ""
+	keyFilePath := ""
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates an gcp account",
```
Comment (jbristowe) [inline pkg/cmd/account/gcp/create/create.go:50]: an ➡️ a

File: pkg/cmd/account/ssh/create/create.go [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+		Ask:     f.Ask,
+		Spinner: f.Spinner(),
+	}
+	descriptionFilePath := ""
+	keyFilePath := ""
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates an ssh account",
+		Long:  "Creates an SSH Account in an instance of Octopus Deploy.",
```
Comment (jbristowe) [inline pkg/cmd/account/ssh/create/create.go:53]: an ➡️ a

**Review by jbristowe — APPROVED** [review]

File: pkg/cmd/account/azure/list/list.go (hunk 4 above)
Comment (domenicsim1) [reply pkg/cmd/account/azure/list/list.go:76]: picking AZURE ENVIRONMENT because ENVIRONMENT already exist on the accounts as Octopus ENVIRONMENTs

**Pull request merged**